SPECS/python-hatchling/python-hatchling.spec

```diff
@@ -33,11 +31,12 @@ BuildRequires:  python3-six
 BuildRequires:  python3-wheel
 BuildRequires:  python3-pathspec
 BuildRequires:  python3-pluggy
-BuildRequires:  python3-tomli
```

Comment:
Is this expected? I see we're removing `python3-toml` since it's now provided by Python 3.12 but I'm not sure about `python3-tomli`.

Reply:
Yes, this is expected. `python-tomli` is no longer needed because Python 3.12 provides `libtoml`. `tomli` and `libtoml` provide equivalent functionality.

Reply:
From the [tomli readme](https://github.com/hukkin/tomli#intro) file on github:

A version of Tomli, the tomllib module, was added to the standard library in Python 3.11 via [PEP 680](https://www.python.org/dev/peps/pep-0680/). Tomli continues to provide a backport on PyPI for Python versions where the standard library module is not available and that have not yet reached their end-of-life.

Reply:
Thank you!